### ISSUE TITLE
FR: Make it possible to set title text, to display in the orange bar at the top.

### ISSUE THREAD

**Issue by SuperElectric** [tensorflow/tensorboard#2015]
### User story
I often have several experiments running in parallel, with a tensorboard tab for each. hen tabbing between them to compare results, it's very hard to keep track of which experiment's results I'm looking at.

### Proposed solution
Allow the user to programmatically set some arbitrary but short (one line) title text, to be displayed in the orange bar at the top of that experiment's tensorboard page.

### Comparison to other feature requests
This request differs from #46 and #194 in that I want a short, human-scannable title at the top, rather than a comprehensive readout of metadata (hyperparams in #46, and metadata in #195). Those are also are good FRs that solve real problems, but they wouldn't serve the purpose here of being a short, readable title, easily scanned by a human when tabbing between experiments.

This also differs from #210, which concerns itself with tagging with metadata, which must be machine-readable, whereas here the focus is on having a human-scannable piece of arbitrary text at a prominent location of the tensorboard page.

**Comment by nfelt** [comment]
This is pretty similar to the `--window_title` command line flag which sets the actual browser window title.  Does that work for you?

**Comment by SuperElectric** [comment]
Huh! I've never noticed a window title on the Chrome browser. Which part of the window does it show up in?

**Comment by nfelt** [comment]
It's just the tab title in Chrome - really by "window" it just means the title of the viewport in the browser.

**Comment by SuperElectric** [comment]
Ah, I see. Unfortunately I think the tab title space will be far too small to fit a descriptive-enough string. For example, the following is a typical experiment description string, taken from the experiment manager I use: 

"pixel-only critics, pixels, control J1, predict all, share both, prediction reward"

...and here's the typical state of my tensorboard tabs:
![tensorboard_tabs_screenshot](https://user-images.githubusercontent.com/5941426/54457073-562d1500-4758-11e9-9222-bbac1fc5c07b.png)

**Comment by abhishek47kashyap** [comment]
> tab title space will be far too small to fit a descriptive-enough string

Second this. Being able to insert a title in the orange bar up top would be great, or anywhere else really where space is not terribly constricted.

**Comment by arcra** [comment]
In case you hadn't thought of this, an easy way to make this work today without any product support, is to use the browser dev tools (e.g. right-click -> "inspect element" on Chrome), and update the white "TensorBoard" text or add an additional div somehow, directly in the html.
